pkg/azuredisk/controllerserver.go

```diff
@@ -1033,6 +1096,31 @@ func (d *Driver) getSnapshotInfo(snapshotID string) (snapshotName, resourceGroup
 	return snapshotName, resourceGroup, subsID, err
 }
 
+// waitForSnapshotCopy wait for copy incremental snapshot to a new region until completionPercent is 100.0
+func (d *Driver) waitForSnapshotCopy(ctx context.Context, subsID, resourceGroup, copySnapshotName string, intervel, timeout time.Duration) error {
+	timeAfter := time.After(timeout)
+	timeTick := time.Tick(intervel)
+
+	for {
+		select {
+		case <-timeTick:
+			copySnapshot, rerr := d.cloud.SnapshotsClient.Get(ctx, subsID, resourceGroup, copySnapshotName)
+			if rerr != nil {
+				return status.Error(codes.Internal, fmt.Sprintf("get snapshot error: %v", rerr.Error()))
```

Comment:
just `return rerr.Error()`

Reply:
I think you could set as 1s as intervel, but print percentage log once every 10s, is that possible? otherwise there would be too many logs.